jekyll/_cci2/first-steps.md

```diff
@@ -9,69 +9,48 @@ version:
+
+4. Use the **Projects** page of the CircleCI app to start building your project code.
+
+## Sign up with GitLab
+{: #gitlab-signup }
+
+1. Click [**Sign Up with GitLab**](https://circleci.com/auth/signup/).
+
+2. Enter your email address, and then set a secure password for your CircleCI account. A verification email is sent to the email address provided.
+
+3. You will be taken to a screen with the option to create a new project from your GitLab repository. Follow the prompts to connect to your GitLab account. Once you have selected a repository and created a new project, you will be redirected to the CircleCI web app dashboard.
+
+The full set of documentation for integrating GitLab with CircleCI can be found on the [GitLab SaaS integration page]({{site.baseurl}}/gitlab-integration).
+{: class="alert alert-info"}
 
 ## Sign up with email
 {: #email-signup }
 
-1. Click **Sign Up with Email**.
-
-    <!-- start: experiment code - #docs-discovery -->
-    <div class="signup-and-try-block">
-      <div class="signup-button-wrapper">
-        <div class="signup-buttons">
-        <a href="https://circleci.com/auth/signup/" class="track-signup-link email-signup-button no-external-icon">
-            <img class="gh-icon" src="{{site.baseurl}}/assets/img/icons/companies/circleci.svg"/>
-            <div class="button-text">Sign Up with Email</div>
-        </a>
-        </div>
-      </div>
-    </div>
-    <!-- end: experiment code -->
+1. Click [**Sign Up with Email**](https://circleci.com/auth/signup/).
 
 2. Enter your email address, and then set a secure password for your CircleCI account. A verification email is sent to the email address provided.
 
-3. Select the options that best describe you and your engineering organization.
+3. If you do not want to connect to your code and only wish to continue with the email signup, click **Cancel**. You will be taken to a screen where you can respond to prompts that best describe your role and your engineering organization.
+
+4. Explore some example projects within the CircleCI app if you do not want to connect to your code at this time. You can take a look at a popular open source project building on CircleCI ([React by Facebook](https://app.circleci.com/pipelines/github/facebook/react)), or one of our own sample projects: a [sample JavaScript app](https://app.circleci.com/pipelines/github/CircleCI-Public/sample-javascript-cfd/), and a [sample Python app](https://app.circleci.com/pipelines/github/CircleCI-Public/sample-python-cfd/).   
 
-4. Connect to your code, or explore some example projects within the CircleCI app if you don't want to connect to your code at this time.
+    You'll be able to start exploring features such as [pipelines]({{site.baseurl}}/pipelines/) and [workflows]({{site.baseurl}}/workflows). The Dashboard, Projects, Organization Settings, and Plan pages are not available until you connect your code.  
```

Comment:
I can't tell if this is supposed to be a part of number 4. in the instructions, but it had a weird indent on it?

```suggestion
You will be able to start exploring features such as [pipelines]({{site.baseurl}}/pipelines/) and [workflows]({{site.baseurl}}/workflows). The Dashboard, Projects, Organization Settings, and Plan pages are not available until you connect your code. 
```